src/config/section/infra/primaryStorages.js

```diff
@@ -61,7 +61,7 @@ export default {
       icon: 'edit',
       label: 'label.edit',
       dataView: true,
-      args: ['tags', 'capacitybytes', 'capacityiops']
+      args: ['name', 'tags', 'capacitybytes', 'capacityiops']
```

Comment:
@davidjumani how does it fare if mgmt server/API does not support the arg?

Reply:
it'll get passed but ignored by the backend, but yes, better add an ACS version check